src/engraving/libmscore/chord.cpp

```diff
@@ -4202,4 +4202,13 @@ void GraceNotesGroup::setPos(double x, double y)
         chord->movePos(PointF(x, y));
     }
 }
+
+Shape GraceNotesGroup::shape() const
+{
+    Shape shape;
+    for (Chord* grace : (std::vector<Chord*>) * this) {
```

Comment:
Is the cast to `(std::vector<Chord*>)` really necessary? For me, it also seems to compile fine without.

Reply:
Huh, first time I tried it didn't work without the cast, I must have done something wrong then. Indeed works fine without